src/CustomElements/v1/CustomElements.js

```diff
@@ -489,8 +489,10 @@ var CustomElementDefinition;
       }
       throw new Error('unknown constructor. Did you call customElements.define()?');
     }
-    patched.prototype = Object.create(orig.prototype);
-    Object.defineProperty(patched.prototype, 'constructor', {value: patched});
+    patched.prototype = Object.create(
+      orig.prototype,
+      {constructor: {configurable: true, value:patched, writable: true}}
```

Comment:
it is indeed, that's what the default is for every property as defined in specs.